features/video-transformation/adaptive-bitrate-streaming.md

```diff
@@ -62,6 +62,16 @@ Use `sr-<representations>` transformation to generate the master manifest file f
 ImageKit uses H.264 codec for encoding video and AAC for encoding audio for both HLS and DASH.
 {% endhint %}
 
+{% hint style="info" %}
+**Important note for existing users**
+
+The ABS output delivered so far ensured a strict 16:9 ratio in output, by adding padding.
+
+Soon we will drop the strict 16:9 ratio, and the output will match the aspect ratio of the original video.
+
```

Comment:
Need to be more specific with dates. 
- so far -> As of DD-MM-YYYY date (date which the doc is published)
- Soon -> In April/May/June 2023

Also, use Grammarly to improve the text. Some basic improvements:
- ensured -> ensures. The behavior has not changed yet, but is about to.
- Soon we will drop the strict 16:9 ratio -> In (month) 2023, this behavior will change such that the aspect ratio of the output video matches the aspect ratio of the original video, instead of strictly being 16:9.